core/issuers.js

```diff
@@ -123,14 +176,16 @@ Description must be truthy: ${description}`;
       // for non-fungible tokens that are unique, destroy them by removing them from
       // the purses/payments that they live in
       mintController.destroy(amount);
+      if (childMint !== undefined) {
+        childMint.destroy(getChildAmount(amount)); // recursively destroy child assets
+      }
     },
     revoke(amount) {
-      this.destroy(amount);
```

Comment:
OMG how did a `this` sneak in? Thanks for removing it!

Separately, let's discuss better linting rules to avoid accidentally introducing `this`.

Reply:
Sure! I think we can have an additional eslint config file that applies the Jessie linting rules and have a separate npm command that does 'npm lint-fix-jessie' or something like that

Reply:
The optional jessie eslint configuration is added